assets/training/model_management/environments/foundation-model-inference/context/Dockerfile

```diff
@@ -1,26 +1,27 @@
-FROM mcr.microsoft.com/aifx/acpt/stable-ubuntu2004-cu117-py38-torch201:{{latest-image-tag:biweekly.\d{6}.\d{1}}}
+FROM mcr.microsoft.com/aifx/acpt/stable-ubuntu2004-cu118-py310-torch201:latest
```

Comment:
Is there a reason to pin the version rather than picking from the template?

Reply:
I'll update that. 
Q: How are the template variables populated? Asking since in the repo, we don't have that populated, so builds won't work (which is why we have to use fixed versions in there). It's buggy to keep changing to template vars when moving code from repo to here.